src/TheThingsNetwork.h

```diff
@@ -23,6 +23,19 @@
 #define HEX_CHAR_TO_NIBBLE(c) ((c >= 'A') ? (c - 'A' + 0x0A) : (c - '0'))
 #define HEX_PAIR_TO_BYTE(h, l) ((HEX_CHAR_TO_NIBBLE(h) << 4) + HEX_CHAR_TO_NIBBLE(l))
 
+/* BEGIN TEST */
+
+#ifdef DEBUG
```

Comment:
I'm going to see if I can put debugPrintLn and debugPrint in the public library.



On Fri, Sep 2, 2016 at 2:32 PM +0200, "Johan Stokking" <notifications@github.com<mailto:notifications@github.com>> wrote:


In src/TheThingsNetwork.h<https://github.com/TheThingsNetwork/arduino-library/pull/12#discussion_r77338168>:

> @@ -23,6 +23,19 @@
>  #define HEX_CHAR_TO_NIBBLE(c) ((c >= 'A') ? (c - 'A' + 0x0A) : (c - '0'))
>  #define HEX_PAIR_TO_BYTE(h, l) ((HEX_CHAR_TO_NIBBLE(h) << 4) + HEX_CHAR_TO_NIBBLE(l))
>
> +/* BEGIN TEST */
> +
> +#ifdef DEBUG


Yes!

-
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/TheThingsNetwork/arduino-library/pull/12/files/8ec238dae563e88bac3f40f9b65259481cb911f3#r77338168>, or mute the thread<https://github.com/notifications/unsubscribe-auth/ATh3kohDtKVO4DuOaoRLK78Y39WBN-8bks5qmBddgaJpZM4Jzms4>.